libdnf/dnf-context.c

```diff
@@ -1420,24 +1419,17 @@ dnf_context_setup_enrollments(DnfContext *context, GError **error)
     if (getuid () != 0)
         return TRUE;
 
-    for (i = 0; cmds[i] != NULL; i++) {
-        int child_argc;
-        g_auto(GStrv) child_argv = NULL;
-        int estatus;
+#ifdef RHSM_SUPPORT
+    g_autoptr(RHSMContext) rhsm_ctx = rhsm_context_new ();
```

Comment:
I should have known that...oh, now I remember what the problem is.  For flatpak+ostree, we need to support Debian stable, which doesn't have it.  But we don't care about Debian stable for libdnf and rpm-ostree.